src/react-apps/applications/ux-editor/src/App.tsx

```diff
@@ -82,6 +82,10 @@ export class App extends React.Component<IAppComponentProps, IAppCompoentState>
       appDataActionDispatcher.fetchLanguage(
         `${altinnWindow.location.origin}/runtime/api/Language/GetLanguageAsJSON`, 'nb');
 
+      // Fetch thirdParty Components
+      appDataActionDispatcher.fetchThirdPartyComponents(
+        `${altinnWindow.location.origin}/runtime/api/resource/${servicePath}/ThirdPartyComponents.json`);
+
```

Comment:
Don't think this is needed in ux-editor.

Reply:
You're right. Deleted.